src/session/secret_chat.rs

```diff
@@ -148,4 +148,8 @@ impl SecretChat {
         self_.state.set(state);
         self.notify("state");
     }
+
+    pub fn formatted_status_expression(user_expression: &gtk::Expression) -> gtk::Expression {
+        User::formatted_status_expression(user_expression)
+    }
```

Comment:
I see that it is called once in `src/session/content/chat_history.rs::129` but do we need this layer of indirection instead of just using `User::formatted_status_expression()` directly?